scripts/nebula.service

```diff
@@ -234,6 +234,10 @@ function status_daemon {
             port=${GREEN}${port}${NC}
         else
             port=${BLINK}${RED}${port}${NC}
+            if [[$daemon_name == nebula-storaged]]; then
+                WARN "${daemon_name} after v3.0.0 will not start service until it is added to cluster."
```

Comment:
will not be started?

Reply:
storaged is started but it hasn't start service. QAQ